main.js

```diff
@@ -1,65 +1,78 @@
 function addToLocalStorage(data) {
-  return localStorage.setItem('books', JSON.stringify(data));
+  return localStorage.setItem("books", JSON.stringify(data));
 }
 
 function getFromLocalStorage(key) {
   return JSON.parse(localStorage.getItem(key));
 }
 
-let collections = [];
-
-function addBookToCollection(title, author) {
-  const id = Math.random();
-  collections.push({ title, author, id });
-  addToLocalStorage(collections);
-  return collections;
+class Book {
+  constructor(title, author, id = Math.random()) {
+    this.title = title;
+    this.author = author;
+    this.id = id;
+  }
 }
 
-function displayBooks() {
-  const getLocalData = getFromLocalStorage('books');
-  collections = getLocalData;
-  const bookSection = document.querySelector('.books-section');
-  bookSection.replaceChildren();
-  collections.forEach((book) => {
-    const singleBook = document.createElement('div');
-    singleBook.innerHTML = `<div>${book.title}</div>
-        <div>${book.author}</div>
-        <button data-id=${book.id} class='remove'>Remove</button>
-        <hr>`;
-    bookSection.appendChild(singleBook);
-  });
-  /* eslint-disable no-use-before-define */
-  removeBookFromCollection();
-}
-function removeBookFromCollection() {
-  const removeBtn = document.querySelectorAll('.remove');
-  removeBtn.forEach((btn) => {
-    btn.addEventListener('click', (e) => {
-      const id = e.target.getAttribute('data-id');
-      collections = collections.filter((el) => el.id !== Number(id));
-      addToLocalStorage(collections);
-      displayBooks();
+class Render {
+  collections = [];
+
+  static addBookToCollection(title, author) {
+    const book = new Book(title, author);
+    this.collections = this.collections.concat(book);
+    console.log(book);
+    addToLocalStorage(this.collections);
+    return this.collections;
+  }
```

Comment:
- [ ] I am not able to add a book please check the error I am getting and fix that
![Screenshot from 2022-10-26 21-28-47](https://user-images.githubusercontent.com/44745855/198118961-7ef07c4b-79a9-40c2-8880-0919a2e08b08.png)

Reply:
Thanks for the feedback :bow: 
While it was working on my machine cos I already had data stored in the local storage, this led to a silent error on any other machine.

With you feedback, I've been able to get it fixed. :+1: